js/ctrls/modal.js

```diff
@@ -91,7 +91,17 @@ angular
             });
         })
         .filter(function(d) { return d.length })
-        .value();
+	.map(function(u){
+          var t=_.toUpper(u[0])
+
+          if(t.startsWith("HTTP")||t.startsWith("MAGNET")||t.startsWith("FTP")){
+            return u
+          }else{
```

Comment:
@xckai can you reverse the if statement? As many un known urls now will become magnet url